src/coreclr/src/vm/reflectioninvocation.cpp

```diff
@@ -2646,7 +2646,7 @@ FCIMPL2(FC_BOOL_RET, ReflectionEnum::InternalEquals, Object *pRefThis, Object* p
 }
 FCIMPLEND
 
-// preform (this & flags) != flags
+// preform (this & flags) == flags
```

Comment:
`preform` does not sound right either. I guess it was meant to be `perform`.

This is very low-value comment. It may be best to delete it.

Reply:
@jkotas, I can change the PR to also correct `preform` or to delete the comment.

Reply:
Yes, that would be nice. I will leave it up to you to pick how to fix this.

Reply:
I kept the comment. This way the number of lines doesn't change. :smile: